### ISSUE TITLE
Whether distributed transaction function will be provided in the future

### ISSUE THREAD

**Issue by JavaLionLi** [polarismesh/polaris#467]
**What is the feature you want to add?**

**Why do you want to add this feature?**

**How to implement this feature?**

**Additional context**

**Comment by andrewshan** [comment]
No, polaris focus on service layer governance, distributed transaction will be supported in another opensource project.
